### ISSUE TITLE
The version in the Find messages might be wrong

### ISSUE THREAD

**Issue by kheaactua** [COVESA/vsomeip#346]
When inspecting with Wireshark, my `OFFER` messages contain the proper version, but my `FIND` messages do not.

_e.g._
```
Offer Service Entry (Service ID 0x0602, Instance ID 0x5679, Version: 1.0)
```

```
Find Service Entry (Service ID 0x0602, Instance ID 0x5679, Version: 1.4294967295)
```

It doesn't seem to matter.  The client knows the interface version is actually 1.0, as `proxy_->getInterfaceVersion()`* is a `Version(1,0)`, and the routing manager will still connect these two.  But it's still suspicious.

\* In addition, in the generated code, my `test_service.hpp` has:
```cpp
45:CommonAPI::Version test_service::getInterfaceVersion() {
46:    return CommonAPI::Version(1, 0);
```
and in `test_serviceSomeIPProxy.cpp` I have
```cpp
125:void test_serviceSomeIPProxy::getOwnVersion(uint16_t& ownVersionMajor, uint16_t& ownVersionMinor) const {
126:    ownVersionMajor = 1;
127:    ownVersionMinor = 0;
```
Basically the version is right everywhere in the code.

**Comment by fcmonteiro** [comment]
Hi,
The 4294967295 (0xFFFF FFFF) in the minor part of the version means any version shall be returned.

![Screenshot from 2023-04-13 10-42-03](https://user-images.githubusercontent.com/129758664/231720733-a89a335a-9c2a-4ac2-9a72-92f52e42267a.png)

cc: @goncaloalmeida

**Comment by kheaactua** [comment]
Yep, I had found that.  I did a local modification in capicxx-someip-runtime to enforce the minor version.  Thanks
